scripts/deleteInactivated.lua

```diff
@@ -148,6 +148,7 @@ local function cleanInactiveUsers(idList)
 end
 
 -- Command body
+redis.replicate_commands();
```

Comment:
makes sense to use that in other scripts?

Reply:
A bit, it stabilizes script behavior for older Redis versions and forces them to use 5.0 version algo. IMHO Lua evaluation and processing on replica nodes bring additional overhead. 

> makes sense to use that in other scripts?

A bit, it stabilizes script behavior for older Redis versions and forces them to use 5.0 version behavior. IMHO Lua evaluation and processing on replica nodes bring additional overhead.